gdeploycore/backend_setup.py

```diff
@@ -198,9 +195,7 @@ def write_thinp_names(self):
                 self.insufficient_param_count('pools', self.device_count)
         for i, j, k in zip(self.pools, self.vgs, self.lvs):
             pools = {}
-            pools['pool'] = i
-            pools['vg'] = j
-            pools['lv'] = k
+            pools['pool'], pools['vg'], pools['lv'] = i, j, k
```

Comment:
@kshithijiyer the first change looks good. But for this one, I prefer the old style of one per line. It is easy to read and change.

Reply:
Done